campaign-launcher/interfaceV2/src/hooks/recording-oracle/campaign.ts

```diff
@@ -117,18 +19,15 @@ type JoinedCampaign = {
   token_symbol: string;
 };
 
-export const useGetUserJoinedCampaigns = (chainId: number | undefined) => {
+export const useGetUserJoinedCampaigns = () => {
   const { isConnected } = useAccount();
-  const { isAuthenticated } = useAuthentication();
+  const { isAuthenticated } = useWeb3Auth();
 
   return useQuery({
-    queryKey: ['user-joined-campaigns', chainId],
-    queryFn: () =>
-      requestWithAuth(`/user/joined-campaigns/${chainId}`, {
-        method: 'GET',
-      }),
-    select: (data) =>
-      data?.map((campaign: JoinedCampaign) => ({
+    queryKey: ['user-joined-campaigns'],
+    queryFn: () => recordingApi.get('/campaigns'),
```

Comment:
Should call `recordingApi.getCampaigns` with proper filter (there is `launcher` filter) instead of filtering on UI